include/polarssl/ecdsa.h

```diff
@@ -78,6 +99,10 @@ int ecdsa_sign( ecp_group *grp, mpi *r, mpi *s,
  * \param blen      Length of buf
```

Comment:
@yanesca One might also rename `buf` and `blen` to `hash` and `hlen` here to emphasize that it's meant to be the hash. Otherwise, it might still happen that a very incautious user will only look at the function signature, see the buffer, len, hash-function combination at the end, and conclude that `buf` will probably be the message to be hashed with the algorithm in the last argument. I'm in general rather opposing to putting too much work into limiting the damage of wrong library usage, but it might be worth it while we're at it.